code/src/App.js

```diff
@@ -1,9 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
+import './index.js'
```

Comment:
I'm not sure but I don't think you need to import `index.js` in the `App.js` component.